index.html

```diff
@@ -67,5 +67,24 @@
       >
     </noscript>
     <!-- built files will be auto injected -->
+    <div id="app"></div>
+
+    <script>
+      ;(function (d, t) {
+        var BASE_URL = "https://chatwoot.incubateur.net"
+        var g = d.createElement(t),
+          s = d.getElementsByTagName(t)[0]
+        g.src = BASE_URL + "/packs/js/sdk.js"
+        g.defer = true
+        g.async = true
+        s.parentNode.insertBefore(g, s)
+        g.onload = function () {
+          window.chatwootSDK.run({
+            websiteToken: "<%= VITE_CHATWOOT_TOKEN %>",
+            baseUrl: BASE_URL,
+          })
+        }
```

Comment:
J'ai trouvé ceci pour paramétrer la position (`left` ou `right`) et un paramètre pour pouvoir cacher la bulle facilement :

```suggestion
        g.onload = function () {
          window.chatwootSettings = {
            hideMessageBubble: false,
            position: 'left',
          };
          window.chatwootSDK.run({
            websiteToken: "<%= VITE_CHATWOOT_TOKEN %>",
            baseUrl: BASE_URL,
            position: "Left",
          })
        }
```

Reply:
Le bouton à gauche est vraiment pas habituel sur grand écran. J'ai préféré coder une mini logique de « responsive » pour avoir le bouton gauche sur téléphone et à droite sinon.

Le bouton ne change pas de place en cas de changement de taille de l'écran mais je ne pense pas que ça soit problématique.

Reply:
Oui, ça suffira largement pour commencer à faire des tests